### DISCUSSION TITLE
修改网页样式的浏览器插件 stylebot

### DISCUSSION THREAD

**Discussion by adoyle-h** [just-talks/tech-talks#58] [3.分享创造]
https://github.com/ankit/stylebot

这个工具的优点是所见即所得，它的样式编辑器类似浏览器开发者工具，直接显示在网页上，可以调整任意网页元素的 CSS 样式。
并且自动保存编辑好的样式，下次打开网页会自动生效。

比如有些网页文档做的比较简陋，看起来费眼睛。

![CleanShot 2023-08-01 at 20 45 01@2x](https://github.com/just-talks/tech-talks/assets/1998490/42a34896-5ed7-4e7d-a9ce-07e9566285cf)

用 stylebot 修改后，相对清楚了一点。

![CleanShot 2023-08-01 at 20 43 35@2x](https://github.com/just-talks/tech-talks/assets/1998490/86bf2260-b3e0-472b-83b3-8f5553e6ebc2)

也可以切换到 CSS 编辑器。

<img width="1104" alt="CleanShot 2023-08-01 at 20 48 15@2x" src="https://github.com/just-talks/tech-talks/assets/1998490/d98c4ca4-c9fe-4443-9d67-788865d9b6f5">

可以随时开关该网页的用户自定义样式。

<img width="300" alt="CleanShot 2023-08-01 at 20 48 15@2x" src="https://github.com/just-talks/tech-talks/assets/1998490/467488c7-525a-40a8-8e4b-1af1c4acd583">

它还支持通配符。比如 url 填 `github.com/just-talks/tech-talks/*`，限定只在匹配这个 url 前缀的网页下应用用户自定义样式。

**Comment by kalxd** [comment]
stylish图形版？不过怎么在上面处理类似于css selector的样式？

```css
body > .content > a.link { ...}

.input:focus {....}
```

**Reply by adoyle-h** [reply]
直接在代码模式写啊
<img width="943" alt="CleanShot 2023-08-03 at 12 21 27@2x" src="https://github.com/just-talks/tech-talks/assets/1998490/b0a5a776-4333-48d1-af96-e5cb37fe24b2">
